src/actions/index.js

```diff
@@ -0,0 +1,14 @@
+import { SIGN_IN, SIGN_OUT } from './types';
+
+export const signIn = (accessToken) => {
```

Comment:
If these actions are specific to GithubAuth compoent then kindly keep these actions in GithubAuth actions file